test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/AbstractService.java

```diff
@@ -240,19 +240,27 @@ private Map<String, Object> getTier2Env() {
         return parseSystemPropertyAsMap("tier2Env");
     }
 
-    private Map<String, Object> parseSystemPropertyAsMap(String systemProperty) {
+    public static Map<String, Object> parseSystemPropertyAsMap(String systemProperty) {
```

Comment:
If this is just for testing, this could have package visibility instead of `public`, right?